.gitignore

```diff
@@ -5,3 +5,5 @@ node_modules
 .netlify
 
 /.idea
+.docker
+.env*
```

Comment:
these ignores .env.example

Reply:
thanks! I'll change this to `.env` only.